identity/src/main/java/com/android/identity/DeviceResponseParser.java

```diff
@@ -435,7 +434,7 @@ void parse(byte[] encodedDeviceResponse,
          * {@link Constants#DEVICE_RESPONSE_STATUS_CBOR_VALIDATION_ERROR}.
          */
         @Constants.DeviceResponseStatus public
-        int getStatus() {
+        long getStatus() {
```

Comment:
This is an API change but I think that's fine.

We should probably change `DEVICE_RESPONSE_STATUS_*` to be `long`, right?

Reply:
I hadn't fixed up all the statuses previously. I think I caught them now.